tsconfig.json

```diff
@@ -8,7 +8,6 @@
     "noEmit": true,
     "isolatedModules": true,
     "strict": true,
-    "moduleResolution": "node",
```

Comment:
@abhay Why did you remove this?

Reply:
because you have to remove it to make it work with the newest tsconfig from react-native

Reply:
Yeah we can just follow what we extend `@react-native/typescript-config/tsconfig.json` which sets it to `"moduleResolution": "bundler"`